src/components/AppMap.vue

```diff
@@ -167,6 +167,7 @@
           </div>
           <div class="col">
             <b-btn size="sm" variant="danger" block @click="drawImport()"><i class="fas fa-file-import"></i> Import</b-btn>
+            <b-form-checkbox id="replace_checkbox" v-model="importReplace" name="replace_checkbox" value=true unchecked-value=false> Replace</b-form-checkbox>
```

Comment:
"Replace" is a bit unclear IMO. How about "Replace existing data" (or something else that makes it clear what is being replaced)?